### ISSUE TITLE
Add Pacific madrone as a primary tree

### ISSUE THREAD

**Issue by almasaeed2010** [statonlab/Treesnap-mobile#535]
Which states would have this tree?

**Comment by almasaeed2010** [comment]
California, Oregon, Washington

**Comment by almasaeed2010** [comment]
[ArbutusMenziesii_TreeSnapNewTreeChecklist_2018_0910_EVC.docx](https://github.com/statonlab/Treesnap-mobile/files/2465833/ArbutusMenziesii_TreeSnapNewTreeChecklist_2018_0910_EVC.docx)

**Comment by jwest60** [comment]
remember to caption the map image https://ppo.puyallup.wsu.edu/pmr/distribution-map/

**Comment by jwest60** [comment]
new questions added for this tree
```js
  standDiversity: {
    label        : 'Stand diversity',
    description  : 'What types of trees do you see in the stand where the tree is found?',
    selectChoices: ['Pure stand of this species', 'Mixed stand (this species and others)', 'Tree is standing alone', 'Not sure'],
    placeHolder  : 'Please select'
  },

  crownAssessment: {
    label        : 'Tree Crown Assessment',
    description  : 'What percentage of the tree crown looks unhealthy or appears damaged?',
    selectChoices: ['< 10%', '10% to 20%', '20% to 50%', '50% to 75%', '> 75%'],
    placeHolder  : 'Please select'
  },

  disease: {
    label        : 'Disease',
    description  : 'Do you see any disease symptoms on the leaves, branches, or trunk?',
    selectChoices: ['Wilting leaves', 'Leaf spots', 'Rust', 'Blight', 'Defoliation', 'Lesions', 'Cankers'],
    placeHolder  : 'Please select',
    multiCheck   : true,
    images       : [
      require('../img/madrone/ARME_wilt-min.jpg'),
      require('../img/madrone/ARME_blight-min.jpg'),
      require('../img/madrone/ARME_dieback-min.jpg'),
      require('../img/madrone/ARME_defoliated-min.jpg'),
      require('../img/madrone/ARME_P_canker-min.jpg'),
      require('../img/madrone/ARME_cankers-min.jpg'),
    ],
    captions    : [
      'Wilting leaves. Marianne Elliott, WSU',
      'Leaf blight (Whole leaf appears dry and dead, brown in color). Marianne Elliott, WSU',
      'Botryosphaeria blight (Leaves that appear grey/black or "burnt"). Marianne Elliott, WSU',
      'Multi-stemmed trunk and/or peeling bark. Michael Yadrick, City of Seattle',
      'Dark, smooth, discolored lesions on trunk. Marianne Elliott, WSU',
      'Raised bumpy cankers on trunk or branches. Michael Yadrick, City of Seattle',
    ]
  }
```

**Comment by jwest60** [comment]
filters have been added for pacific madrone on the treesnap website

**Comment by almasaeed2010** [comment]
`disease` was changed to `madroneDisease` on both the app and the site.

**Comment by almasaeed2010** [comment]
Released in 1.6.0
